### PR TITLE
Description component improve

### PR DESCRIPTION
**Pull request by DmitiryPotychkin**
- stop event bubbling after click on component's buttons
- add ability to pass content in component, display it in line with description text

### CODE REVIEW

File: projects/components/src/description/description.component.ts [hunk 1]
```diff
@@ -21,18 +21,19 @@ import {
         data-sensitive-pii
         #eventDescriptionText
       >
+        <ng-content></ng-content>
```
Comment (aaron-steinfeld) [inline projects/components/src/description/description.component.ts:24]: I'm not against using content to pass in the description, but it's an alternative API to passing in description as an input - we shouldn't have both. If we have use cases that need more than a simple string passed in, let's make this the sole API to pass in content and remove the `description` input.

Comment (DmitiryPotychkin) [reply projects/components/src/description/description.component.ts:23]: Fixed

**Review by aaron-steinfeld — COMMENTED** [review]
lgtm - will let @anandtiwary take a pass too

File: projects/components/src/description/description.component.ts (hunk 1 above)
Comment (anandtiwary) [reply projects/components/src/description/description.component.ts:23]: Curious to know why we need ng-content? Is their an example usage I could see? Is it suppose to wrap any other component apart from a string?

Comment (aaron-steinfeld) [reply projects/components/src/description/description.component.ts:23]: I think the main use cases are links and formatted strings

Comment (anandtiwary) [reply projects/components/src/description/description.component.ts:23]: okay.. Links we should handle from outside, but i see the need for formatted strings.

**Review by anandtiwary — APPROVED** [review]

**Review by aaron-steinfeld — APPROVED** [review]

**Pull request merged**